swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/agents/ScheduledAgent.kt

```diff
@@ -69,11 +70,19 @@ abstract class ScheduledAgent(
   @PostConstruct
   private fun init() {
     log.info("Initializing agent ${javaClass.simpleName}")
-    registry.gauge(lastRunAgeId.withTag("agentName", javaClass.simpleName), this, {
+    registry.gauge(lastRunAgeId.withTag("agentName", javaClass.simpleName), this) {
       Duration
         .between(it.getLastAgentRun(), clock.instant())
         .toMillis().toDouble()
-    })
+    }
+  }
+
+  @PreDestroy
+  private fun stop() {
+    log.info("Shutting swabbie down")
```

Comment:
I think this log should be changed to something like, `.info("Scheduled action {} is stopping", javaClass.simpleName)`

"Shutting swabbie down" sounds scary and isn't really accurate.